### DISCUSSION TITLE
Add example of how to implement an actions cell

### DISCUSSION THREAD

**Discussion by italodeandra** [icflorescu/mantine-datatable#43] [Ideas]
Sometimes we want to have some actions for each row on the table and not use the dropdown menu.

I'm currently using the dropdown menu because I didn't have time to think of an alternative.

**Comment by otobot1** [comment]
I'm not sure what you mean by an `actions cell`. If you simply want an action to happen when the row is clicked, you can use the [`onRowClick` property](https://icflorescu.github.io/mantine-datatable/examples/handling-row-clicks).

**Reply by italodeandra** [reply]
`onRowClick` only handles one action, which usually is to "open".

"Actions cell" is that cell you have on every row which contains several buttons with different actions.

**Reply by icflorescu** [reply]
I think @italodeandra refers to a cell containing icons for operations like `view`, `edit`, `delete`, etc.

**Reply by icflorescu** [reply]
Added issue https://github.com/icflorescu/mantine-datatable/issues/47.

@italodeandra, if you wanna give it a try, make sure to work on `next` branch.

I've created the example stub; relevant files are:
- `docs/pages/examples/row-actions-cell.tsx`
- `/docs/examples/RowActionsCellExample.tsx`

**Comment by icflorescu** [comment]
It's done.

**Reply by italodeandra** [reply]
Thanks!
